ormin/queries.nim

```diff
@@ -37,6 +39,79 @@ var
     Function(name: "replace", arity: 3, typ: dbVarchar)
```

Comment:
Be nice to upgrade these at some point to also support type checking the args. But alas, not today.